pad_server/src/state.h

```diff
@@ -12,10 +14,16 @@
 typedef struct {
     bool actuators[NUM_ACTUATORS];
     arm_lvl_e arm_level;
-    pthread_mutex_t lock;
+
+    pthread_rwlock_t rw_lock;
+    pthread_rwlockattr_t rw_lock_attr;
```

Comment:
I don't believe there's a need to store the attributes since it's never used past the initialization of the lock.